MediaWikiKit/MediaWikiKit/MWKSectionList.m

```diff
@@ -140,4 +140,37 @@ - (NSArray*)entries {
     return self.sections;
 }
 
+#pragma mark - Hierarchy
+
+- (void)buildSectionHierarchy {
```

Comment:
To directly answer your question: it might be able to be called at the right point(s), but I'm not 100% sure it should be